constants.bzl

```diff
@@ -3,6 +3,11 @@ RELEASE_ENGINES_NODE = "^18.19.1 || ^20.11.1 || >=22.0.0"
 RELEASE_ENGINES_NPM = "^6.11.0 || ^7.5.6 || >=8.0.0"
 RELEASE_ENGINES_YARN = ">= 1.13.0"
 
+NG_PACKAGR_VERSION = "^19.1.0-next.0"
+ANGULAR_VERSION = "^19.1.0-next.0"
```

Comment:
Question: What exactly does `ANGULAR_VERSION` represent? The `@angular/core` / FW version used for direct dependencies? I wonder if a more specific name like `ANGULAR_FW_VERSION` might be helpful to communicate its purpose.